node-playwright-webkit/Dockerfile

```diff
@@ -0,0 +1,364 @@
+FROM debian:bookworm AS build
+
+ARG NODE_VERSION=22.8.0
+
+RUN set -xe; \
+    apt-get -yqq update; \
+    apt-get -yqq install \
+        libatk1.0 \
+        libatk-bridge2.0 \
+        libatomic1 \
+        libcairo-gobject2 \
+        libenchant-2.2 \
+        libepoxy0 \
+        libevent-2.1 \
+        libflite1 \
+        libgdk-pixbuf-2.0 \
+        libgles2 \
+        libgstreamer1.0-0 \
+        libgstreamer1.0-dev \
+        libgstreamer-gl1.0-0 \
+        libgstreamer-plugins-bad1.0-0 \
+        libgstreamer-plugins-base1.0-0 \
+        libgtk-3-0 \
+        libharfbuzz-icu0 \
+        libhyphen0 \
+        liblcms2-2 \
+        libmanette-0.2-0 \
+        libopus0 \
+        libpango-1.0 \
+        libpangocairo-1.0 \
+        libsecret-1-0 \
+        libtinfo6 \
+        libvpx7 \
+        libwebpdemux2 \
+        libwebpmux3 \
+        libwoff1 \
+        libx264-164 \
+        libx264-dev \
+        libxslt1.1 \
+    ; \
+    apt-get -yqq install \
+        curl \
+    ;
+
+RUN set -xe; \
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash; \
+    . ~/.bashrc; \
+    nvm install ${NODE_VERSION} \
+    ;
+
+WORKDIR /app
+COPY package* .
+
+RUN set -xe; \
+    . ~/.bashrc; \
+    npm install \
+    ;
+
+# Node binary
+RUN set -xe; \
+    cp /root/.nvm/versions/node/v${NODE_VERSION}/bin/node /usr/bin/node \
+    ;
+
+RUN mkdir /home/tmp
+
+FROM scratch
+
+ARG NODE_VERSION=22.8.0
+
+# Create required directories.
+COPY --from=build /home/tmp /tmp
+
+# Browser binary
+COPY --from=build /root/.cache/ms-playwright /root/.cache/ms-playwright
+
+# Browser libraries
+COPY --from=build \
+                  /usr/lib/x86_64-linux-gnu/libabsl_base.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_debugging_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_demangle_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_graphcycles_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_int128.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_malloc_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_raw_logging_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_spinlock_wait.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_stacktrace.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_strings_internal.so.20220623 \
+                  /usr/lib/x86_64-linux-gnu/libabsl_strings.so.20220623 \
```

Comment:
How many of these are from node, and how many are specific to playwright? Asking because if most are from Node, again, using the docker image for it might be easier

Reply:
All of those are for WebKit / Playwright.

Reply:
Not sure, again, why using a Node Docker image would be easier. I would still use `FROM scratch` to extract only the required bits.

Reply:
Ok, so all libraries for playwright, understood